### DISCUSSION TITLE
Seek and Destroy, waypoint height adjustment, load and deliver

### DISCUSSION THREAD

**Discussion by ODoyle040** [Courseplay/Courseplay_FS22#692] [Ideas]
First post and i didn't take a whole lot of time to search all the discussions so i apologize if this has already been discussed in advance. 
quick and dirty list of ideas:
1) Seek and destroy function: I understand this may be way overreaching and even a bit lazy on my part but i thought it might be a cool function if its even possible. Make a "clean up" or "seek and destroy" function where the desired equipment finishes its course and then can check the field for missed spots in the instance of say, plowing, or missed crops or windrows of hay/grass. Another aspect ive thought about with this is when i lose a course or forget to save it and i have to start over on a half finished field so the equipment skips over finished areas of the field and lifts the implement and follows the course till it reaches the crop or unfinished work in its new course. It may be overly complicated to code for or something, idk. It was just an idea and you never know till you suggest it.

2) waypoint height adjust: self explanatory. i just want to set them lower so i can pan over the equipment and see whats going on easier without having to turn them off.

3) Load and deliver is a great function! i love it. its a bit buggy but after using courseplay since it came out in fs19 i know your group always continues to improve and fix their mod and i appreciates you for it. adding load and deliver functionality to combines and crop harvesters would be most excellent.

Last but not least i always have a difficult time setting up following. i dont know if its even possible to make it easier to set your harvesters up with a trailer following the pipe, like you see on irl farm videos, but ive never had great success at making this function work well. it would be even cooler if it was set up with load and deliver function. 

Again, my apologies for any ignorance i may be displaying here. I really appreciate your mod and wouldnt play farm simulator nearly as long or as much without your great work. 

Thank you!

**Comment by Redster999** [comment]
1. Would be nice, but I dont believe that how courseplay work. But perhaps later we can create a "border" and then only run the generate field on the border. That was possible in fs19 and might come back
2. Pretty fast to turn them on and off via the new hud though. Just click on the vehicle. But my guess this will come also in another hud update. just a trigger in the hud needed
3. CP now works together with Autodrive, Autodrive can unload your combine and dump it in the silo already. + its editable and much more configurable. But the load fertilizer/seed  whilst using CP is coming on another version.

The last part should be dooable already. Just adjust the follower to start at 0%. I dont know how good that will work.

**Reply by Redster999** [reply]
Check out Autodrive and youtube is your friend here.

**Reply by ODoyle040** [reply]
thank you! i know i need to sit down and be patient and watch some youtube videos on this stuff. i always try and figure it out myself and get about halfway there. i recall having AD in fs19 and forgot about it till now, so ill be downloading it today. appreciate ya

**Comment by Tensuko** [comment]
1. Won't do, way to complicated.
2. Visual lowering very very low priority if we even consider that. Turn it off and done, but I can also tell you that the HUD won't get expanded much further.
3. We won't make course recording to drive on them and also no trigger functions. That is AD part and we conenct those mods as good as both teams can do it. No reason to develope the same things twice.

**Reply by ODoyle040** [reply]
what about being able to adjust the unload distance for say a beet harvester? they always stop a little too far from the trailer. is there an easy fix for that? sorry im getting on youtube now but since i have your attention i figured id ask.

**Reply by ODoyle040** [reply]
i appreciate your quick response, understand and accept what youre telling me btw. i understood that the first one was way too complicated but figured id ask. thank you

**Reply by Redster999** [reply]
Yes there is a setting autodrive for the distance between the combine and the follower.

**Reply by ODoyle040** [reply]
ty redster ill get that autodrive now and get back to my farm management!

**Reply by Redster999** [reply]
Nice. Once you get it you will love it and understand why this was a good idea

**Comment by ODoyle040** [comment]
in addition, i never realized how important autodrive was to courseplay as in they compliment eachother. awesome job on that end, i love it when mods work together well

**Reply by Tensuko** [reply]
Thank you, that was part of the plan for FS22 since many ppl used both in FS19 already.
And AutoDrive also improoved already and will improove further.
So try it out ;)
